daemon/cmld.c

```diff
@@ -153,6 +161,35 @@ cmld_container_get_by_uuid(uuid_t *uuid)
 	return NULL;
 }
 
+
+
+// TODO: check where to insert usb_serial_short into list
+container_t *
+cmld_container_get_by_usb_serial_short(char *usb_serial_short)
+{
+	ASSERT(usb_serial_short);
+
+	for (list_t *l = cmld_containers_list; l; l = l->next)
+	{
+		for (list_t *dev = container_get_usbdev_list(l->data); dev; dev = dev->next)
+		{
+			if (strncmp(uevent_usbdev_get_i_serial(dev->data),usb_serial_short,SERIAL_SHORT_LENGTH))
+			{
+				DEBUG("[+] Found container for usb serial: '%s'. Checking for USBDEV_TYPE ('%p)'", usb_serial_short, dev->data);
```

Comment:
plz remove [+] to keep the message style consistent
also, remove "Checking for USBEV_TYPE...."

Reply:
Removed "[+]" and unnecessary DEBUG prints everywhere